lib/building.rb

```diff
@@ -0,0 +1,52 @@
+class Building
+  attr_reader :units, :renters, :rented_units
+
+  def initialize
+    @units = []
+    @renters = []
+    @rented_units = []
+  end
+
+  def add_unit(unit)
+    @units.push(unit)
+  end
+
+  def add_renters
+    @units.each do |unit|
+      @renters << unit.renter.name
+    end
+  end
+
+  def average_rent
+    (@units[0].monthly_rent + @units[1].monthly_rent).to_f / 2
+  end
+
+  def add_rented_units
+    @rented_units << @units[1]
+  #   @units.each do |unit|
+  #     if unit.renter != nil
+  #       @rented_units << unit
+  #   end
+  # end
```

Comment:
We should figure this out by going through the units and seeing which has a renter